<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IVAppWidget</class>
 <widget class="QWidget" name="IVAppWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeView" name="objectsView"/>
         <widget class="shared::ui::ObjectsTreeView" name="importView"/>
         <widget class="shared::ui::ObjectsTreeView" name="sharedView"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="shared::ActionsBar" name="toolBar" native="true"/>
       </item>
       <item>
        <widget class="ive::GraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>shared::ui::ObjectsTreeView</class>
   <extends>QTreeView</extends>
   <header>interface/objectstreeview.h</header>
  </customwidget>
  <customwidget>
   <class>shared::ActionsBar</class>
   <extends>QWidget</extends>
   <header>actionsbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ive::GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>itemeditor/graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
